server/api.go

```diff
@@ -169,7 +172,7 @@ func (p *Plugin) submitScore(w http.ResponseWriter, r *http.Request) {
 
 	// Thank the user for their feedback when they first answer the survey
 	if isFirstResponse {
-		p.CreateBotDMPost(userID, p.buildFeedbackRequestPost())
+		_, _ = p.CreateBotDMPost(userID, p.buildFeedbackRequestPost())
```

Comment:
Please log the error

Reply:
okey, i will add that

Reply:
i fix this in this [4a8672](https://github.com/mattermost/mattermost-plugin-nps/pull/79/commits/4a8672f349b2bc34a822d17e42dfc25ace0212ed)